xbmc/guilib/GUIAudioManager.cpp

```diff
@@ -160,64 +164,35 @@ void CGUIAudioManager::PlayPythonSound(const std::string& strFileName, bool useC
     return;
 
   // If we already loaded the sound, just play it
-  pythonSoundsMap::iterator itsb=m_pythonSounds.find(strFileName);
+  const auto& itsb = m_pythonSounds.find(strFileName);
   if (itsb != m_pythonSounds.end())
   {
-    IAESound* sound = itsb->second;
+    const auto& sound = itsb->second;
     if (useCached)
     {
       sound->Play();
       return;
     }
     else
     {
-      FreeSoundAllUsage(sound);
       m_pythonSounds.erase(itsb);
     }
   }
 
-  IAESound *sound = LoadSound(strFileName);
+  const auto& sound = LoadSound(strFileName);
```

Comment:
no ref

Reply:
Yeah, in this case move semantics saves us the reference counting anyway :+1: